osc/obs_scm/project.py

```diff
@@ -199,6 +199,8 @@ def status(self, pac: str):
         if st is None and exists:
             return '?'
         elif st is None:
+            if os.path.exists(os.path.join(self.absdir, '.git')):
+                return '?'
```

Comment:
This is confusing.
`?` indicates a file that is not tracked the current OBS SCM checkout.
Now you return this value even if it is tracked under a **different** SCM (git).

Reply:
solved different now